packages/codegen-ui-react/lib/forms/form-renderer-helper/cta-props.ts

```diff
@@ -166,7 +166,7 @@ const getRecordUpdateDataStoreCallExpression = ({
 
 /**
   Object.entries(modelFields).forEach(([key, value]) => {
-    if (typeof value === 'string' && value.trim() === "") {
+    if (typeof value === 'string' && value === "") {
       modelFields[key] = undefined;
```

Comment:
To my knowledge, need to update this comment as well?

Reply:
ah yeah, I'll update that comment on a future commit.